rocAL/include/api/rocal_api_augmentation.h

```diff
@@ -132,6 +132,31 @@ extern "C" RocalTensor ROCAL_API_CALL rocalCropResize(RocalContext context, Roca
                                                       RocalTensorLayout output_layout = ROCAL_NONE,
                                                       RocalTensorOutputType output_datatype = ROCAL_UINT8);
 
+/*! \brief Fused function which perrforms crop and resize on images.
```

Comment:
Check the spelling of performs, and it is a random function right, so we need to modify the functions accordingly